### ISSUE TITLE
Develop a JavaScript Tic-Tac-Toe app.

### ISSUE THREAD

**Issue by Ratheshan03** [Ratheshan03/Small-JavaScript-Projects#4]
**### Requirements:**

- Create a Tic Tac Toe for 2 Players.
- Show scores of the Players.

Bonuses 🎉🎉:

- Make it for One Player.
- Show win or Loose on the set of 5 games.

Any number of people can work on a single issue,
This issue is open to all.

Check the CONTRIBUTION.md before making your contributions 
Happy Coding 🎃 👩🏻‍💻

**Comment by Maximun3213** [comment]
Can i join this issue ?

**Comment by Ratheshan03** [comment]
@Maximun3213  Yes I can assign you! Please make sure you check the CONTRIBUTIONS.md
